src/components/farm-tokens/farm-tokens.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import { View, Text } from 'react-native';
+
+import { Divider } from 'src/components/divider/divider';
+import { TokenIcon } from 'src/components/token-icon/token-icon';
+import { FarmToken } from 'src/interfaces/earn.interface';
+import { formatSize } from 'src/styles/format-size';
+import { conditionalStyle } from 'src/utils/conditional-style';
+import { getTruncatedProps } from 'src/utils/style.util';
+
+import { useFarmTokensStyles } from './farm-tokens.styles';
+
+interface Props {
+  stakeTokens: Array<FarmToken>;
+  rewardToken: FarmToken;
+}
+
+const TOKENS_SYMBOLS_DIVIDER = ' / ';
+
+export const FarmTokens: FC<Props> = ({ stakeTokens, rewardToken }) => {
+  const styles = useFarmTokensStyles();
+
+  return (
+    <View>
+      <View style={styles.row}>
+        <View style={[styles.row, styles.tokensContainer]}>
+          {stakeTokens.map((token, index) => (
```

Comment:
is it ok that we use `.map` instead of `<Flatlist />` ?

Reply:
It's okay exactly here because the amount of tokens is pretty small